publisher_scaled_joint_trajectory_controller:
  ros__parameters:

    controller_name: "scaled_joint_trajectory_controller"
    wait_sec_between_publish: 6

    goal_names: ["pos1", "pos2", "pos3", "pos4", "pos5"]
    pos1:
      positions: [-1.5708, -2.5, 2.5, -3.1415, 0.0, 0.0]
    pos2:
      positions: [1.5053, -1.2618, 1.8317, -2.1601, -1.5708, -0.0654]
    pos3:
      positions: [1.5053, -1.2095, 1.8841, -2.2383, -1.5708, -0.0654]
    pos4:
      positions: [-0.1003, -1.2401, 1.8501, -2.1901, -1.5708, -0.0436]
    pos5:
      positions: [0.0, -2.1817, 2.1817, -1.5708, -1.5708, 0.0]
   
    joints:
      - shoulder_pan_joint
      - shoulder_lift_joint
      - elbow_joint
      - wrist_1_joint
      - wrist_2_joint
      - wrist_3_joint

    check_starting_point: true
    starting_point_limits:
      shoulder_pan_joint: [-0.1,0.1]
      shoulder_lift_joint: [-1.6,-1.5]
      elbow_joint: [-0.1,0.1]
      wrist_1_joint: [-1.6,-1.5]
      wrist_2_joint: [-0.1,0.1]
      wrist_3_joint: [-0.1,0.1]

publisher_joint_trajectory_controller:
  ros__parameters:

    controller_name: "joint_trajectory_controller"
    wait_sec_between_publish: 6

    goal_names: ["pos1", "pos2", "pos3", "pos4", "pos5"]
    pos1:
      positions: [-1.5708, -2.5, 2.5, -3.1415, 0.0, 0.0]
    pos2:
      positions: [1.5053, -1.2618, 1.8317, -2.1601, -1.5708, -0.0654]
    pos3:
      positions: [1.5053, -1.2095, 1.8841, -2.2383, -1.5708, -0.0654]
    pos4:
      positions: [1.5053, -1.2618, 1.8317, -2.1601, -1.5708, -0.0654]
    pos5:
      positions: [-0.1003, -1.2401, 1.8501, -2.1901, -1.5708, -0.0436]
    pos6:
      positions: [0.0, -2.1817, 2.1817, -1.5708, -1.5708, 0.0]
    pos7:
      positions: [-0.1003, -1.1921, 1.8588, -2.2381, -1.5708, -0.0436]
    pos8:
      positions: [-1.5708, -2.5, 2.5, -3.1415, 0.0, 0.0]

  

    joints:
      - shoulder_pan_joint
      - shoulder_lift_joint
      - elbow_joint
      - wrist_1_joint
      - wrist_2_joint
      - wrist_3_joint

    check_starting_point: true
    starting_point_limits:
      shoulder_pan_joint: [-0.1,0.1]
      shoulder_lift_joint: [-1.6,-1.5]
      elbow_joint: [-0.1,0.1]
      wrist_1_joint: [-1.6,-1.5]
      wrist_2_joint: [-0.1,0.1]
      wrist_3_joint: [-0.1,0.1]
